src/main/java/Calculator.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.*;
+
+public class Calculator {
+
+    static int splitAndSum(String inputData) {
+        if (checkIfEmpty(inputData)) {
+            return 0;
+        }
+
+        List<String> elements = splitText(inputData);
+
+        return sumAll(elements);
+    }
+
+    private static boolean checkIfEmpty(String text) {
+        return  text == null || text.isEmpty();
+    }
+
+    private static List<String> splitText (String inputData){
+        String delimiter, realText;
+
+        final Matcher m = Pattern.compile("//(.)\n(.*)").matcher(inputData);
+        if (m.find()) {
+            delimiter = m.group(1);
+            realText = m.group(2);
+        } else {
+            delimiter = ",|:";
+            realText = inputData;
+        }
+
+        return Arrays.asList(realText.split(delimiter));
+    }
+
+    private static int sumAll (List<String> strings) {
+        return convertStringArrayToIntArray(strings).stream().mapToInt(Integer::intValue).sum();
```

Comment:
```suggestion
        return convertStringsToNumbers(strings).stream()
            .mapToInt(Integer::intValue)
            .sum();
```
반환값도 변경되었으니 메서드명도 같이 변경하면 좋을 것 같네요 😄